src/pygama/raw/build_raw.py

```diff
@@ -66,6 +68,13 @@ def build_raw(
     overwrite
         sets whether to overwrite the output file(s) if it (they) already exist.
 
+    config_file
+        Specification of config file, used for decoding CoMPASS files
+
+        - if None, CompassDecoder will eat the first packet to determine waveform length
+        - if a str ending in ``.json``, interpreted as a filename containing
+          json-shorthand for the output specification (see :mod:`.compass.compass_event_decoder`).
+
```

Comment:
Why not merging this extra config section in `out_spec`? Would avoid us having to add a new argument to `build_raw`.